extensions/wikia/AdEngine/js/AdEngineRunner.js

```diff
@@ -9,12 +10,23 @@ define('ext.wikia.adEngine.adEngineRunner', [
 	require.optional('ext.wikia.adEngine.lookup.prebid'),
 	require.optional('ext.wikia.aRecoveryEngine.adBlockRecovery'),
 	require.optional('wikia.articleVideo.featuredVideo.lagger')
-], function (adEngine, adTracker, instantGlobals, log, win, a9, prebid, adBlockRecovery, fvLagger) {
+], function (adContext, adEngine, adTracker, instantGlobals, log, win, a9, prebid, adBlockRecovery, fvLagger) {
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.adEngineRunner',
 		supportedModules = [a9, prebid, adBlockRecovery, fvLagger],
-		timeout = instantGlobals.wgAdDriverDelayTimeout || 2000;
+		timeout = getTimeout();
+
+	function getTimeout() {
+		var fvTimeout;
+
+		if (fvLagger && fvLagger.wasCalled() && adContext.get('targeting.hasFeaturedVideo')) {
```

Comment:
FV is already checked in `fvLagger.wasCalled()`:
https://github.com/Wikia/app/pull/15039/files#diff-149b9f80140f2da67453bf8168d872d8R21